<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Encoding</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="7" column="1" >
    <widget class="QLabel" name="labelQz" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLabel" name="labelAudCodec" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" >
    <widget class="QLabel" name="label_17" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Total Size</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_11" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Vid Bitrate</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" >
    <widget class="QLabel" name="label_16" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Audio Size</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="label_13" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Quantizer</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" >
    <widget class="QLabel" name="labelContainer" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_9" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>FPS</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="labelFrame" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Frame</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="labelPhasis" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Phasis</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Aud Codec</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLabel" name="labelVidCodec" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" >
    <widget class="QLabel" name="label_15" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Video Size</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" >
    <widget class="QLabel" name="labelVideoSize" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Vid Codec</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" >
    <widget class="QLabel" name="labelETA" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" >
    <widget class="QLabel" name="label_18" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>ETA</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" >
    <widget class="QPushButton" name="pushButton" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QLabel" name="labelFps" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" >
    <widget class="QLabel" name="label_19" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Container</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" >
    <widget class="QLabel" name="labelTotalSize" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" >
    <widget class="QLabel" name="labelAudioSize" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QLabel" name="labelVidBitrate" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_25" >
     <property name="font" >
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Encoding file</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
